examples/simulation/pitop_simulate.py

```diff
@@ -2,8 +2,6 @@
 
 use_virtual_hardware()
 
-from time import sleep  # noqa: E402
-
 from pitop import Pitop  # noqa: E402
 from pitop.pma import LED, Button  # noqa: E402
```

Comment:
I think these should go to the top of the file... unless `use_virtual_hardware` is needed at the very top of the file?

Reply:
yeah component imports have to come after `use_virtual_hardware` so that they use the mocks - i've added a comment at least. users aren't really meant to have call `use_virtual_hardware` so this shouldn't cause friction normally (they'd use real hardware or it's mocked during import on a cloud server with the env var PITOP_VIRTUAL_HARDWARE)